ansible-scylla-monitoring/tasks/docker.yml

```diff
@@ -68,6 +74,9 @@
         -R {{ scylla_monitoring_prom_rules_path }}/ \
         -b --storage.tsdb.retention.time={{ scylla_monitoring_retention }} \
         {{ scylla_monitoring_params | default('') }}
+  environment:
+    SCYLLA_USER: "{% if monitoring_cql_username is defined and monitoring_cql_password is defined %}{{ monitoring_cql_username }}{% else %}{% endif %}"
```

Comment:
Do you really need an `else` clause here and below?

Reply:
Nope. Fixed.